### PR TITLE
Fix issue in 3.x star args function signatures

### LINKED ISSUE #268: Python 3.x star args function signatures
The following fails in at least Python 3.2-3.7

```
def test_11(*args, name):
    return args, name

assert test_11(1, 2, 3, name='hi') == ((1, 2, 3), 'hi'), "*args name"

def test_12(a, *args, name):
    return a, args, name

assert test_11(1, 2, 3, name='hi') == ((1, 2, 3), 'hi'), "*args, name"
assert test_12(1, 2, 3, name='hi') == (1, (2, 3), 'hi'), "a, *args, name"
```

We get for example: 

```
    def test_11(*args, ):

```

The problem is in `make_function.py`, `build_param()` again. 

@x0ret - if you are not up for this, lemme know and I will look at.

### PR DESCRIPTION
**Pull request by x0ret**
Fixes #268 

[This issue](https://github.com/rocky/python-decompile3/pull/15#discussion_r294685638) is also fixed.

@rocky, I also passed previous test cases, however i'd be grateful if you review.

### CODE REVIEW

File: uncompyle6/semantics/make_function.py [hunk 1]
```diff
@@ -764,52 +758,34 @@ def build_param(ast, name, default, annotation=None):
...
-            for n in node:
-                if n == 'pos_arg':
-                    continue
-                elif self.version >= 3.4 and not (n.kind in ('kwargs', 'no_kwargs', 'kwarg')):
-                    continue
-                else:
-                    self.preorder(n)
-                    ends_in_comma = False
-                break
+        if 3.0 <= self.version <= 3.5:
```
Comment (x0ret) [inline uncompyle6/semantics/make_function.py]: I went through versions and came to this that for <= 3.5 all can be handles using following proc.
Please let me know if you have comment on this. Thanks.

**Comment by x0ret** [comment]
All new test cases i used:

```python
def test_10(*, name):
    return args, name

def test_11(*args, name):
    return args, name

def test_13(*args, name: int=1, qname):
    return args, name

def test_14(*args, name='S', fname, qname=4):
    return args, name

def test_12(a, *args, name):
    return a, args, name
```

File: uncompyle6/semantics/make_function.py (hunk 1 above)
Comment (rocky) [reply uncompyle6/semantics/make_function.py]: No comments other than admiration and praise. Thanks for simplifying this.

**Comment by rocky** [comment]
@x0ret would you merge this into decompyle3 please? I tried doing but it didn't work out.

**Comment by x0ret** [comment]
> @x0ret would you merge this into decompyle3 please? I tried doing but it didn't work out.

Yes, sure.

**Pull request merged**